### ISSUE TITLE
Potential false positives with `font-variation-settings`

### ISSUE THREAD

**Issue by scurker** [dequelabs/axe-core#3305]
Product: axe-core `4.3.5`

Expectation: axe-core should not raise a false positive when setting [`font-variation-settings`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variation-settings)

Actual: A false positive is raised

Motivation: No false positives

The following CSS creates a `14 point` font with a `900 weight`:

```css
.font-variation-setting {
  background-color: #fff;
  color: #878787;
  font-size: 14px;
  font-variation-settings: "wght" 900, "wdth" 200;
}
```

For 2 elements, one with and one without `font-variation-settings` looks like the following:

![two 14px fonts, the one with font-variation-settings bold, the one without normal](https://user-images.githubusercontent.com/1062039/142674015-b3e86f8c-d570-47e0-8f3a-fa27dbbd8272.png)

[Related Codepen](https://cdpn.io/scurker/debug/BadMPjd/PBkNWRozyDbM)

It appears `window.getComputedStyle(...).fontWeight` returns the font weight of the inherited `font-weight` value and not the one from `font-variation-settings`. Checking the value from `fontVariationSettings` in computed style would likely resolve this issue.
